.github/workflows/integrate.yaml

```diff
@@ -70,15 +72,22 @@ jobs:
 
       - name: Run test
         run: |
+          sg microk8s -c "microk8s enable rbac ingress metallb:10.64.140.43-10.64.140.49"
           # Requires the model to be called kubeflow due to this bug:
           # https://github.com/kubeflow/kubeflow/issues/6136
           juju add-model kubeflow
-          sudo apt install -y firefox-geckodriver
+          wget https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb
+          sudo apt install ./google-chrome-stable_current_amd64.deb
+          sudo apt-get install chromium-chromedriver
```

Comment:
Can we separate this into its own step so it is clear what is environment setup and what is test?

Reply:
Thanks